templates/index.mustache

```diff
@@ -16,10 +16,7 @@
 </head>
 
 <body>
```

Comment:
Just a question about this file 
Do we need the old content of this index file?